posthog/batch_exports/http.py

```diff
@@ -276,3 +284,46 @@ def perform_destroy(self, instance: BatchExport):
         temporal = sync_connect()
         delete_schedule(temporal, str(instance.pk))
         instance.save()
+
+
+class BatchExportLogEntrySerializer(DataclassSerializer):
+    class Meta:
+        dataclass = BatchExportLogEntry
+
+
+class BatchExportLogViewSet(StructuredViewSetMixin, mixins.ListModelMixin, viewsets.GenericViewSet):
```

Comment:
I'd make this one be generic too for any log fetching, is there a reason not to?